src/olympia/addons/admin.py

```diff
@@ -103,12 +117,11 @@ def version__is_blocked(self, obj):
 
     version__is_blocked.short_description = 'Block status'
 
-    def hash_link(self, obj):
-        url = reverse('zadmin.recalc_hash', args=(obj.id,))
-        template = '<a href="{}" class="recalc" title="{}">Recalc Hash</a>'
-        return format_html(template, url, obj.hash)
```

Comment:
If this is the only place `recalc_hash` is referenced (and I think it is) we should remove that view + associated tests and js

Reply:
Removed in https://github.com/mozilla/addons-server/pull/20244/commits/f49220dcb0fc74a9893096119c8d37e57473366d